featurebyte/service/task_manager.py

```diff
@@ -123,6 +123,45 @@ async def get_task(self, task_id: str) -> Task | None:
             progress=document.get("progress"),
         )
 
+    async def update_task_result(self, task_id: str, result: Any) -> None:
+        """
+        Update task result
+
+        Parameters
+        ----------
+        task_id: str
+            Task ID
+        result: Any
+            Task result
+        """
+        await self.persistent.update_one(
+            collection_name=TaskModel.collection_name(),
+            query_filter={"_id": task_id},
+            update={"$set": {"task_result": result}},
+            user_id=self.user.id,
+        )
+
+    async def get_task_result(self, task_id: str) -> Any:
+        """
+        Get task result
+
+        Parameters
+        ----------
+        task_id: str
+            Task ID
+
+        Returns
+        -------
+        Any
+            Task result
+        """
+        document = await self.persistent.find_one(
+            collection_name=TaskModel.collection_name(),
+            query_filter={"_id": task_id},
```

Comment:
Is the type of `task_id` str or ObjectId?

Reply:
![Screenshot 2024-02-27 at 4 38 13 PM](https://github.com/featurebyte/featurebyte/assets/37915749/b290ebfe-7b90-4a3d-a681-8fc35423603c)

The expected type is string.